tests/Functional/BreadcrumbRenderTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+/**
+ * Tests that breadcrumbs are rendered correctly.
+ *
+ * @group batch1
+ */
+class BreadcrumbRenderTest extends ContentRenderTestBase {
```

Comment:
We are not testing the rendering here, but the breadcrumb cache, so its behaviour. Let's name this simply BreadcrumbTest.